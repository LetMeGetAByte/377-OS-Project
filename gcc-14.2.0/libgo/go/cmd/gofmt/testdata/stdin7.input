version https://git-lfs.github.com/spec/v1
oid sha256:1e694a05e22bddb28c1a3b1d020b892d6e98089e53efd4a007af7f3d532410d9
size 296
