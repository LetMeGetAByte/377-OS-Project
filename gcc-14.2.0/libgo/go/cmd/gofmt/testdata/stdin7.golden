version https://git-lfs.github.com/spec/v1
oid sha256:3619aa99684725060fb524944cc0995f08f99d3b0cd851461756355dda6b95e2
size 342
